### PR TITLE
fix(client): fix ecma version check worker error

### PR DESCRIPTION
**Pull request by easy1090**
## Summary
Fix ecma version check worker error.
When click some file on loader analysis page, will have page error.
img_v3_026i_e67ded7e-e6bd-4a41-9b16-30c3dfb82c1g

## Checklist

- [x] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/components/src/components/worker/jsequal/client.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Tag } from 'antd';
 import { master } from '../../../utils/worker';
 import { useI18n } from '../../../utils';
 
-const getJSEqualWorker = () => new Worker(new URL('./worker.ts', import.meta.url));
+const getJSEqualWorker = () => new Worker(new URL( './worker.js', import.meta.url));
```
Comment (chenjiahan) [inline packages/components/src/components/worker/jsequal/client.tsx:6]: Extra space

```suggestion
const getJSEqualWorker = () => new Worker(new URL('./worker.js', import.meta.url));
```

File: packages/components/src/components/worker/ecmaversion/client.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { BadgeColorMap, BadgeType } from '../../Badge';
 import { ECMAVersion } from '@rsdoctor/utils/ruleUtils';
 import { master } from 'src/utils/worker';
 
-const getECMAVersionDetectWorker = () => new Worker(new URL('./worker.ts', import.meta.url));
+const getECMAVersionDetectWorker = () => new Worker(new URL('./worker.js', import.meta.url));
```
Comment (chenjiahan) [inline packages/components/src/components/worker/ecmaversion/client.tsx:6]: Why do we need to use .js for worker, does module-tools support worker.ts?

Comment (easy1090) [reply packages/components/src/components/worker/ecmaversion/client.tsx:6]: yes, this './worker.ts' can not compile to .js, so show use .js.

Comment (chenjiahan) [reply packages/components/src/components/worker/ecmaversion/client.tsx:6]: @10Derozan cc, can we support this scenario

**Review by chenjiahan — APPROVED** [review]

File: packages/components/src/components/worker/ecmaversion/client.tsx (hunk 2 above)
Comment (10Derozan) [reply packages/components/src/components/worker/ecmaversion/client.tsx:6]: Yes, we recommand use .js when it is a worker.

Comment (chenjiahan) [reply packages/components/src/components/worker/ecmaversion/client.tsx:6]: @10Derozan I do not find the document for this

**Pull request merged**